packages/atomic-angular/angular.json

```diff
@@ -35,5 +35,8 @@
       }
     }
   },
-  "defaultProject": "@coveo/atomic-angular"
+  "defaultProject": "@coveo/atomic-angular",
+  "cli": {
+    "analytics": false
+  }
```

Comment:
At some point, angular prompted me to choose whether I wanted analytics or not, and it automatically appended that line.

It's not the first time it does this, so I'm including it in this PR once and for all.